nservicebus/msmq/distributor/index.md

```diff
@@ -230,9 +95,4 @@ Check out [John Breakwell's blog](https://blogs.msdn.microsoft.com/johnbreakwell
 
 If the Distributor goes down, even if its worker nodes remain running, they do not receive any messages. It is important to run the Distributor on a cluster that has its its queues configured as clustered resources.
 
-Since the Distributor performs no CPU or memory intensive work, several Distributor processes can be placed on the same clustered server. Be aware that the network IO may end up being the bottleneck for the Distributor, so take into account message sizes and throughput when sizing the infrastructure.
-
-
-## Licensing
-
-partial: licensing
+The distributor is disk and network IO restricted. If a single endpoint does not saturate either of these then it is possible to host multiple distributor processes on a single server.
```

Comment:
Nope, distributors. Because assuming that we're distributing big chinks of work (this is stated above), distributor should do relatively little work -- we can co-host distributors for multiple endpoints on a single machine/cluster.